worker/src/unstract/worker/worker_interface.py

```diff
@@ -0,0 +1,77 @@
+from abc import ABC, abstractmethod
+
+
+class UnstratcWorkerInterface(ABC):
+    @abstractmethod
+    def __init__(self, image_name: str, image_tag: str):
+        pass
+
+    @abstractmethod
+    def _get_image(self) -> str:
+        pass
+
+    @abstractmethod
+    def _image_exists(self, image_name_with_tag: str) -> bool:
+        pass
```

Comment:
I don't think these 2 needs to be part of interface. This is only required in docker_worker